Documentation/Books/Manual/Deployment/README.md

```diff
@@ -21,3 +21,8 @@ By _Technology_:
 - [Docker](Docker/README.md)
 - [Kubernetes](Kubernetes/README.md)
 - [Mesos, DC/OS](DCOS/README.md)
+
+By _Cloud Provider_:
```

Comment:
IMO "By" is the wrong word here. 
We already have "By deployment" & "By technology" and both are still applicable when you go in the cloud. The topics described under here are  purely an intro and referral to various "By technology" items.

So consider calling it "In the cloud"